### PR TITLE
FLUT-6523 - [Others] Fixed the issues raised in incidents

### PR DESCRIPTION
**Pull request by Marismathan**
## Description ##

- Changed the maximum zoom level as 0.3 in this section https://help.syncfusion.com/flutter/cartesian-charts/zoom-pan#maximum-zoom-level. (Related forum link: https://support.syncfusion.com/agent/tickets/343944)
- Added new topic for adding image in legend icon.
(Related forum link: https://support.syncfusion.com/agent/tickets/375936)

### CODE REVIEW

File: Flutter/cartesian-charts/legend.md [hunk 1]
```diff
@@ -110,6 +110,59 @@ The [`name`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
 
 ![Customized Legend](images/legend/customized_legend.png)
 
+## Customizing legend icon as image
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/legend.md:113]: make as 3rd level heading

File: Flutter/cartesian-charts/legend.md [hunk 2]
```diff
@@ -110,6 +110,59 @@ The [`name`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
...
+
+An image is also used as legend icon by using the [`legendItemBuilder`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/Legend/legendItemBuilder.html). If an image is used as legend icon, it should be added in the project and referred as asset in pubspec file.
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/legend.md:115]: already we have explained builder property, I have asked to explain the legendIcon type for image

**Review by dharanidharandharmasivam — CHANGES_REQUESTED** [review]

File: Flutter/cartesian-charts/legend.md (hunk 1 above)
Comment (Marismathan) [reply Flutter/cartesian-charts/legend.md:113]: Added this as third heading

File: Flutter/cartesian-charts/legend.md (hunk 2 above)
Comment (Marismathan) [reply Flutter/cartesian-charts/legend.md:115]: Changed the code snippets and content as we discussed

File: Flutter/cartesian-charts/legend.md [hunk 3]
```diff
@@ -110,6 +110,49 @@ The [`name`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
 
 ![Customized Legend](images/legend/customized_legend.png)
 
+### Legend icon with image
+
+By providing the needed image in [`image`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/Legend/image.html) property of [`legend`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/Legend-class.html) and the [`legendIconType`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartSeries/legendIconType.html) as `LegendIconType.image` in [`series`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartSeries-class.html), an image can also be utilised as a legend icon. If an image is used as a legend icon, it should be placed in its own folder in the project, and that folder should be referenced in the pubspec file's asset.
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/legend.md:115]: The legend icon shape can be changed using the [`legendIconType`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartSeries/legendIconType.html) of the [`ChartSeries`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartSeries-class.html). To use the legend icon as image, specify the [`legendIconType`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/ChartSeries/legendIconType.html) as `LegendIconType.image`, add the image using the [`image`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/Legend/image.html) property of [`legend`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/Legend-class.html). Ensure that the image or image folder is included as an asset in the pubspec.yaml file.

**Review by dharanidharandharmasivam — CHANGES_REQUESTED** [review]

File: Flutter/cartesian-charts/legend.md (hunk 3 above)
Comment (Marismathan) [reply Flutter/cartesian-charts/legend.md]: Changed the description as mentioned

**Review by dharanidharandharmasivam — APPROVED** [review]
Hi @JayavigneshwaranG ,

The content is fine and you can merge this.

Thanks,
Dharani.

**Pull request merged**